### PR TITLE
Add place dialog

### PR DESCRIPTION
**Pull request by opyh**
Adds a dialog that shows how to add PoIs on Wheelmap.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
 import MainView, { UnstyledMainView } from './MainView';
 
 import type {
   Feature,
-  WheelmapFeature,
-  AccessibilityCloudFeature,
-  YesNoLimitedUnknown,
-  YesNoUnknown,
-  NodeProperties,
+    WheelmapFeature,
```
Comment (mutaphysis) [inline src/App.js]: Unneeded indentation? (Also lines below)

File: src/MainView.js [hunk 2]
```diff
@@ -12,20 +13,25 @@ import NotFound from './components/NotFound/NotFound';
...
 import colors from './lib/colors';
+import { isFirstStart } from './lib/savedState';
 import { hasBigViewport, isOnSmallViewport } from './lib/ViewportSize';
 
 import type {
   Feature,
-  YesNoLimitedUnknown,
-  YesNoUnknown,
-  NodeProperties,
+    YesNoLimitedUnknown,
```
Comment (mutaphysis) [inline src/MainView.js]: Unneeded indentation? (Also lines below)

**Comment by mutaphysis** [comment]
A bit hard to review, contains quite some stuff from other branches  - also seems to conflict still

**Comment by opyh** [comment]
Better? :)

File: src/components/MainMenu/MainMenu.js [hunk 3]
```diff
@@ -89,35 +88,52 @@ class MainMenu extends React.Component<Props, State> {
   }
 
   setupFocusTrap() {
-    if (this.homeLink && this.addPlaceLink) {
-      this.homeLink.addEventListener('keydown', this.focusToLastElement);
-      this.addPlaceLink.addEventListener('keydown', this.focusToFirstElement);
```
Comment (opyh) [inline src/components/MainMenu/MainMenu.js]: This and the other stuff below just fixes several type errors that popped up in this file.

**Pull request merged**